### PR TITLE
Gdr 2224

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Related JIRA issue: GDR-2224

## Why was it changed?
To fix bug with displaying drug_moa and drug_moa2 for single-agent data and extending list of headers

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [X] Package version bumped
- [X] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Review by gladkia — APPROVED** [review]

File: R/standardize_MAE.R [hunk 1]
```diff
@@ -198,13 +198,14 @@ get_optional_coldata_fields <- function(se) {
 #'
 get_optional_rowdata_fields <- function(se) {
```
Comment (j-smola) [inline R/standardize_MAE.R:199]: Maybe you could add?
```
checkmate::assert_class(se, "SummarizedExperiment")
```

File: tests/testthat/test-headers.R [hunk 2]
```diff
@@ -5,7 +5,7 @@ test_that("get_header works", {
   expect_equal(get_header("manifest"), list(barcode = c("Barcode", "Plate"),
                                             template = c("Template", "Treatment"),
                                             duration = "Duration"))
-  expect_equal(length(get_header()), 13)
+  expect_equal(length(get_header()), 14)
```
Comment (j-smola) [inline tests/testthat/test-headers.R:8]:
```suggestion
  expect_equal(length(get_header()), 16)
```

```
> get_header() |> names()
 [1] "manifest"             "raw_data"             "normalized_results"   "averaged_results"     "response_metrics"     "metric_average_filds"
 [7] "add_clid"             "metrics_names"        "metrics_results"      "controlled"           "reserved"             "ordered_1"           
[13] "ordered_2"            "id"                   "combo"                "obsolete"      
```

**Review by j-smola — COMMENTED** [review]
Could you maybe add some test for `get_optional_rowdata_fields()`?

File: tests/testthat/test-headers.R (hunk 2 above)
Comment (bczech) [reply tests/testthat/test-headers.R]: Thanks, done

File: R/standardize_MAE.R (hunk 1 above)
Comment (bczech) [reply R/standardize_MAE.R:200]: done

Comment (j-smola) [reply R/standardize_MAE.R:200]: ✔️

File: R/utils.R [hunk 3]
```diff
@@ -409,3 +409,39 @@ geometric_mean <- function(x, fixed = TRUE, maxlog10Concentration = 1) {
...
+#' @param dt data.table with Metric data
+#' @param fixed Flag should be add fix for -Inf in geometric mean.
+#' @param average_fields Character vector of column names in \code{dt} to take 
+#' the average of.
+#' @param geometric_average_fields Character vector of column names in \code{dt} 
+#' to take the geometric average of.
+#' 
+#' @return data.table without replicates
+#' @export
+average_biological_replicates_dt <- function(
```
Comment (j-smola) [inline R/utils.R:425]: Coudl you add some tests for this function? 👀

File: R/manage_additional_metadata.R [hunk 4]
```diff
@@ -0,0 +1,193 @@
...
+  } else {
+    x[additional_var] <- NULL
+  }
+  # remove special class
+  class(x) <- setdiff(class(x), additional_var_names)
+  return(x)
+}
+
+#' @keywords internal
+modify_label <- function(x, option, var_name) {
```
Comment (j-smola) [inline R/manage_additional_metadata.R:185]: This is an internal function, but maybe you could add tests to show how this function is supposed to work?

**Review by j-smola — APPROVED** [review]

File: R/utils.R (hunk 3 above)
Comment (j-smola) [reply R/utils.R:425]: ✔️

**Pull request merged**